docs/start-stop.rst

```diff
@@ -117,3 +117,32 @@ with a README documenting the installation instructions, is available
 within the source repository at ``util/scripts/lsfcsm``.
 
 Support for the SLURM resource manager is under development.
+
+-----------------------------------------------
+  Stage-in and Stage-out Manifest File Format
+-----------------------------------------------
+
+The --stage-in and --stage-out options for unify (start and terminate,
+respectively) require a manifest file to specifiy the files to move.
+The manifest file is a one-for-one list of files to be copied from and copied
+to.  Each file to be copied is fully specified on one line of the
+manifest file.  There are no directories or wild cards; each file
+is specified separately.
+
+Each line of a manifest file consists of the source file, then some white
```

Comment:
"Each line is formatted as `<source file> <whitespace> <destination file>`" Note that there is no need to explicitly talk about the newline character in the format - it is implied in the fact that it's a line.